netcore/build.sh

```diff
@@ -61,24 +84,32 @@ while [[ $# > 0 ]]; do
 done
 
 
-cd ..
-./autogen.sh --with-core=only
-cd netcore
+# run .././autogen.sh only once or if "--rebuild" argument is provided
+if [[ "$force_rebuild" == "true" || ! -f .configured ]]; then
+  cd ..
+  ./autogen.sh --with-core=only
+  cd netcore
+  touch .configured
```

Comment:
Should we isolate in the netcore folder?  I don't have an opinion other than that.

Reply:
it's currently inside the netcore folder

Reply:
facepalm :-)